geomesa-cassandra/geomesa-cassandra-tools/src/main/scala/org/locationtech/geomesa/tools/cassandra/commands/CommandWithCassandraDataStore.scala

```diff
@@ -0,0 +1,35 @@
+package org.locationtech.geomesa.tools.cassandra.commands
+
+import org.geotools.data.DataStoreFinder
+import org.locationtech.geomesa.cassandra.data.{CassandraDataStore, CassandraDataStoreParams}
+import org.locationtech.geomesa.tools.cassandra.CassandraConnectionParams
+import org.locationtech.geomesa.tools.common.commands.CommandWithDataStore
+
+
+trait CommandWithCassandraDataStore extends CommandWithDataStore {
+  val params: CassandraConnectionParams
+  lazy val ds = CassandraDataStoreParamsHelper.createDataStore(params)
+}
+
+
+object CassandraDataStoreParamsHelper {
+
+  def getDataStoreParams(params: CassandraConnectionParams): Map[String, String] = {
+    Map[String, String](
+      CassandraDataStoreParams.CONTACT_POINT.getName -> params.contactPoint,
+      CassandraDataStoreParams.KEYSPACE.getName -> params.keySpace,
+
+      CassandraDataStoreParams.NAMESPACE.getName -> "blah")
```

Comment:
we have to keep the namespace param - geoserver requires it for ContentDataStores